ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/RemoteRepoTimeoutNeverDisableTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2013 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.ftest.core.content;
+
+import org.apache.http.HttpStatus;
+import org.commonjava.indy.client.core.IndyClientException;
+import org.commonjava.indy.core.change.StoreEnablementManager;
+import org.commonjava.indy.ftest.core.AbstractContentManagementTest;
+import org.commonjava.indy.ftest.core.category.TimingDependent;
+import org.commonjava.indy.model.core.Group;
+import org.commonjava.indy.model.core.RemoteRepository;
+import org.commonjava.maven.galley.model.Location;
+import org.commonjava.test.http.expect.ExpectationServer;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.commonjava.indy.model.core.StoreType.remote;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class RemoteRepoTimeoutNeverDisableTest
```

Comment:
I know we don't have this on a lot of our tests, but for these tests you're writing / modifying, could you provide a class-level comment that describes the preconditions, actions, and assertions of the test, plus a small one- or two-sentence summary of what the test checks?

We need to start improving this part of our code a bit, and at least preventing it from getting worse is a good first step IMO.